### DISCUSSION TITLE
Browser tab is hanging during writing comments in the issues and doing PR reviews

### DISCUSSION THREAD

**Discussion by binarymaster** [community/community#62276] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I start to write something in the issues comments or reviewing a PR, the browser tab may start hanging for 20-30 seconds. This slowdowns the review process a lot, and is barely tolerable.

The problem started to happen since 21 July or so, and it looks like this:

![image](https://github.com/community/community/assets/578406/2e67545f-4e2c-49d6-b225-0e4888d6b8a5)

This error is being spammed in the browser console, which could be the root of the problem:

```
Uncaught (in promise) DOMException: Failed to execute 'matches' on 'Element': ':popover-open' is not a valid selector.
    at ToolTipElement.handleEvent (https://github.githubassets.com/assets/vendors-node_modules_primer_view-components_app_components_primer_primer_js-node_modules_gith-e7a1e2-0c753046063c.js:208:1830)
```

![image](https://github.com/community/community/assets/578406/a8a65063-405e-4b56-8302-35ebe921050b)

Performance analysis:

![image](https://github.com/community/community/assets/578406/5b44a3d1-324e-4cab-ade6-0ceb6f97f33d)

----------------------

I'm using Chrome version 109.0.5414.120, this is the last one available for Windows 7, and I cannot afford a more powerful machine or an OS upgrade right now.

**Comment by nirav-gajera** [comment]
1. **Update your browser.** This error may have been fixed in a newer version of your browser.
2. **Disable your extensions.** Sometimes extensions can interfere with the normal functionality of a browser. Try disabling your extensions and see if that fixes the issue.
3. **Clear your browser's cache and cookies.** This can sometimes help to resolve issues with a browser.

If the issue is still not resolved, you can try to contact GitHub support for help. : https://github.com/contact

**Reply by binarymaster** [reply]
Hi,

Thanks for your answer. I have tried your suggestions, and it didn't helped.

1. I'm using latest browser version available for the OS.
2. I've disabled all browser extensions.
3. I've logged out from my account, cleared out the cache and cookies, and logged back again.

None of these actions helped, I'm still experiencing the problem. It also can be reproduced in the incognito mode.

**Reply by nirav-gajera** [reply]
contact at github support by creating ticket they'll help you : https://support.github.com/
